### ISSUE TITLE
Error finding meeting topic if contains quote

### ISSUE THREAD

**Issue by yosizelensky** [Battleman/zoomdl#19]
- [X] I have updated to the latest version,
- [X] (If applicable) I have ensured the password is correct with the flag `-v 0`,
- [X] I have searched the opened issues for a duplicate,

**Describe the bug**
If the meeting's topic contains a quote, like `This isn't going to work`, the regex that collects the `meta`s won't be able to find it, resulting in the script failing:

```
Traceback (most recent call last):
  File "zoom_dl\__main__.py", line 20, in <module>
  File "zoom_dl\__init__.py", line 18, in main
  File "zoom_dl\zoomdl.py", line 148, in download
  File "zoom_dl\zoomdl.py", line 89, in download_vid
AttributeError: 'NoneType' object has no attribute 'replace'
[29036] Failed to execute script __main__
```

**To Reproduce**
1. Have a meeting containing a quote for the topic
2. Try to download
3. Fail

**Expected behavior**
The `meta`s regex should be able to collect the topic if it contains quotes.

**Versions (please complete the following information):**
 - OS: Ubuntu 20.04, Windows 10
 - Python Version: 3.8.7

**Additional context**
I fixed it for myself by simply adding a `try`/`except` around the faulting `replace` to fallback to a generic meeting name, but the regex itself should be fixed.

**Comment by Battleman** [comment]
Very good point. I changed the way the metadata is extracted, with a more robust regexp and the use of Yaml to have a good dictionary. This should be fixed in v2020.09.08 (released later today)

**Comment by Battleman** [comment]
Fixed in 2020.09.08
